Makefile

```diff
@@ -36,3 +36,8 @@ test-pipelines:
 test-pyefi:
 	pytest lib/pyEFI
 
+tests-clean:
+	rm -rf /tmp/nextflow/efi_tests/
+	rm -rf .nextflow/
```

Comment:
Keep directory

Reply:
As you suggested in slack, I could make multiple cleaning recipes in the Makefile. W/ the `mktemp` process that I developed, a `make tests-clean` call would only need to  happen very rarely if at all. So adding more cleaning recipes to the Makefile doesn't seem necessary.

Reply:
From discussion https://github.com/EnzymeFunctionInitiative/EST/issues/64#issuecomment-2442120159. 
> 4. running a `make` command that works on an external hard-coded directory seems like a bad practice

I will update the `tests-clean` recipe to only remove repo-local files.